lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -1963,6 +1967,9 @@ bool TFFunctionPartition::markFunction(bool &hasTensorOps) {
       loggedInput = true;
     }
   };
+  
+  // Split all critical edges for block marking.
+  splitAllCondBrCriticalEdgesWithNonTrivialArgs(hostFn, &DI, &LI);
```

Comment:
Please add a fixme to remove this when TFPartition is integrated into deabstraction, thanks!

Reply:
Addressed in #18437.